packages/fxa-graphql-api/src/database/database.service.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { setupAuthDatabase, setupProfileDatabase } from 'fxa-shared/db';
+import { Account } from 'fxa-shared/db/models/auth';
+import Knex from 'knex';
+
+import { AppConfig } from '../config';
+
+@Injectable()
+export class DatabaseService {
+  public authKnex: Knex;
+  public profileKnex: Knex;
+
+  constructor(configService: ConfigService<AppConfig>) {
+    const dbConfig = configService.get('database') as AppConfig['database'];
+    this.authKnex = setupAuthDatabase(dbConfig.mysql.auth);
+    this.profileKnex = setupProfileDatabase(dbConfig.mysql.profile);
+  }
+
+  async dbHealthCheck(): Promise<Record<string, any>> {
+    let status = 'ok';
+    try {
+      await Account.query().limit(1);
+    } catch (err) {
```

Comment:
Should this error get kicked over to Sentry?

Reply:
It didn't before, this part hasn't changed in the NestJS transition. We catch them in other places to always return a value so this was following existing code. We could file a follow-up issue if we'd like to catch these as well.